model/jpa/src/main/resources/default-persistence.xml

```diff
@@ -19,7 +19,7 @@
              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
              xsi:schemaLocation="https://jakarta.ee/xml/ns/persistence https://jakarta.ee/xml/ns/persistence/persistence_3_0.xsd"
              version="3.0">
-    <persistence-unit name="keycloak-default" transaction-type="RESOURCE_LOCAL">
+    <persistence-unit name="keycloak-default" transaction-type="JTA">
```

Comment:
I need to verify the behavior for Undertow, as there's no/limited support for JTA (AFAIK). Probably would be better to explicitly change the TX type in the codebase for Quarkus.

Reply:
+1 for keeping RESOURCE_LOCAL with Undertow for this PR. We have plans to change that eventually, but it needs more investigation. Can you override this setting on-the-fly for Quarkus only as you suggest in your previous comment? I think this would be my preferred solution.

Reply:
Done